config/application.yml

```diff
@@ -0,0 +1,17 @@
+defaults: &defaults
+  jwt:
+    key: <%= Rails.root.join('config', 'jwt', 'dev.pem') %>
```

Comment:
Configuration for production should be added. Private key should no t be committed. We should copy it into the place where atmo will be running. Take a look at https://gitlab.com/eurvalve/vapor/blob/master/config/application.yml#L21-25 and vapor installation for examples.